google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/flexible/AppEngineFlexibleDeploymentEditor.java

```diff
@@ -0,0 +1,549 @@
+/*
```

Comment:
@patflynn @etanshaul seems like you guys are skipping this file? (You need to click "show diff" to see it)